src/main/java/org/embulk/filter/column/ColumnVisitorImpl.java

```diff
@@ -244,13 +245,19 @@ public void jsonColumn(Column outputColumn)
                 pageBuilder.setNull(outputColumn);
             }
             else {
-                String jsonPath = new StringBuilder("$.").append(outputColumn.getName()).toString();
+                String name = outputColumn.getName();
+                // Strips any of a set of characters from the start and end of a String.
```

Comment:
Why strip? It will make different column name? Why not escape? Is there no escape specification for json path bracket?